components/browser/engine-gecko-beta/src/main/java/mozilla/components/browser/engine/gecko/GeckoEngineSession.kt

```diff
@@ -132,8 +132,14 @@ class GeckoEngineSession(
     /**
      * See [EngineSession.reload]
      */
-    override fun reload() {
-        geckoSession.reload()
+    override fun reload(bypassCache: Boolean) {
```

Comment:
Instead of passing a `boolean`, I think we should pass our `LoadUrlFlags`: https://github.com/mozilla-mobile/android-components/blob/master/components/concept/engine/src/main/java/mozilla/components/concept/engine/EngineSession.kt#L388

And default to "none".  That's what we do in `loadUrl`: https://github.com/mozilla-mobile/android-components/blob/master/components/concept/engine/src/main/java/mozilla/components/concept/engine/EngineSession.kt#L425

Reply:
ah yes! Made the change 👍